packages/fxa-auth-server/test/mocks.js

```diff
@@ -779,3 +780,13 @@ function mockVerificationReminders(data = {}) {
     ),
   };
 }
+
+function mockStripeHelper(methods) {
+  //const StripeHelper = require('../lib/payments/stripe').StripeHelper;
+  //const log = module.exports.mockLog();
+  //const config = {
+  //  subhub: {plansCacheTtlSeconds: 1000}
+  //};
```

Comment:
Missed comments?

Reply:
Oh, all of those are me working on tests mocking. This is a piecemeal pr, trying to break things down into smaller hunks 😓